dataactvalidator/filestreaming/loadFile.py

```diff
@@ -10,15 +11,39 @@ def loadCgac(filename):
 def loadObjectClass(filename):
     LoaderUtils.loadCsv(filename,ObjectClass,ValidatorValidationInterface(),{"max_oc_code":"object_class_code","max_object_class_name":"object_class_name"},{"object_class_code":{"skip_duplicates":True}})
 
+def loadSF133(filename,skipClear = False):
+    """ Load SF133 files, set skipClear to True for second file """
+    LoaderUtils.loadCsv(filename,SF133,ValidatorValidationInterface(),{"ata":"allocationtransferagencyidentifier","aid":"agencyidentifier","availability_type_code":"availabilitytypecode","bpoa":"beginningperiodofavailability","epoa":"endingperiodofavailability","main_account":"mainaccountcode","sub_account":"subaccountcode","fiscal_year":"fiscal_year","period":"period","line_num":"line","amount_summed":"amount"},{"allocationtransferagencyidentifier":{"pad_to_length":3},"agencyidentifier":{"pad_to_length":3},"mainaccountcode":{"pad_to_length":4},"subaccountcode":{"pad_to_length":3}},skipClear=skipClear)
+
 def loadProgramActivity(filename):
     LoaderUtils.loadCsv(filename, ProgramActivity, ValidatorValidationInterface(), {"year":"budget_year","agency_id":"agency_id",
         "alloc_id":"allocation_transfer_id","account":"account_number","pa_code":"program_activity_code","pa_name":"program_activity_name"},
         {"program_activity_code":{"pad_to_length":4},"agency_id":{"pad_to_length":3},"account_number":{"pad_to_length":4},"allocation_transfer_id":{"pad_to_length":3}})
 
-def loadDomainValues(basePath):
+
+def loadDomainValues(basePath, localSFPath = None):
+    """ Load all domain value files, localSFPath is used to point to a SF-133 file, if not provided it will be downloaded from S3  """
```

Comment:
I guess that last sentence is what I'm questioning. Our current model is to load domain values from the validator's config file, which we can't do for the full version because S3. From my perspective as a local user, I'd rather the broker look in the config file for a local version because that's where the other domain files are located.

Does that make sense?